pisek/jobs/cache.py

```diff
@@ -27,6 +27,11 @@
 SAVED_LAST_SIGNATURES = 5
 
 
+class NoAliasDumper(yaml.Dumper):
+    def ignore_aliases(self, data):
+        return True
```

Comment:
Why do we need this?

Reply:
Because we export `CacheEntries` one by one. This will always generate alias with id 1. And ids have to be unique.